pancake.txt

```diff
@@ -0,0 +1,24 @@
+Pancake Recipe
+
+Ingredients
+1 1/2 cup flour
+2 teaspoon baking powder
+1 teaspoon baking soda
+1/2 salt
+1 cup of milk
+1 egg
+1 tablespoon olive oil
+
```

Comment:
can we have butter instead of olive oil?

Reply:
yes

Reply:
thanks!